android/src/main/java/com/amplitude/android/Amplitude.kt

```diff
@@ -155,5 +155,43 @@ open class Amplitude(
 fun Amplitude(apiKey: String, context: Context, configs: Configuration.() -> Unit): com.amplitude.android.Amplitude {
     val config = Configuration(apiKey, context)
     configs.invoke(config)
-    return com.amplitude.android.Amplitude(config)
+    @Suppress("DEPRECATION")
+    val modifiedConfig = Configuration(
```

Comment:
Any changes made by the function passed to this function to add defaultTracking options will be ignored unless a new `Configuration` is created from the modified configuration.